test/features/datarequest.feature

```diff
@@ -0,0 +1,102 @@
+@smoke
+Feature: Datarequest
+
+# DQL-26: Data Requests - Install and configure extensionUAT - the Data Requests are accessible via the /datarequest URL (does not display a 404 error)
```

Comment:
here and below - please remove all references to internal ticket numbers - it does not make sense to keep internal references in open source project. Also, these comments become stale very fast.

As for the scenario title - please use whatever is in the comment as a Scenario title.

Reply:
Scenario titles are should be as long as possible and describe exactly the behaviour its trying to accomplish.
i.e.
# DQL-27: Data Base & Form updatesUAT - when visiting the /datarequests page as a non-logged in user, the button at the top of the page reads "Login to create a data request" and links to the /user/login page
    Scenario: View 'Login to create data request' button when not logged in

should be
Scenario: Given an anonymous user visits, when they visit a datarequest page, Then a button is displayed at the top of the page with the text "Login to create a data request" and when clicked redirects the user to the Login Page (/user/login)